docs/management/managing-saved-objects.asciidoc

```diff
@@ -39,6 +39,17 @@ have multiple environments for development and production.
 Import and export also work well when you have a large number
 of objects to update and want to batch the process.
 
+[float]
+==== Compatibility across versions and downgrades
```

Comment:
As you removed the content about downgrades, should the title be "Compatibility across versions"?